tools/flang2/flang2exe/CMakeLists.txt

```diff
@@ -162,7 +167,7 @@ target_include_directories(flang2
 
 # Install flang2 executable
 install(TARGETS flang2
-  RUNTIME DESTINATION bin)
+        RUNTIME DESTINATION ${LLVM_TOOLS_BINARY_DIR})
```

Comment:
@xoviat Why was this line ever needed? It makes the destination directory different for `flang1` and `flang2`, and seems to have caused a build issue for some users (#1338).